test/rule-matches/aria-dpub-role-fallback-matches.js

```diff
@@ -0,0 +1,97 @@
+describe('aria-dpub-role-fallback-matches', function () {
```

Comment:
Your "role" tests are probably a bit too specific here. This should test that the code works. What this exact list of roles is shouldn't matter as much. The integration tests should solve that for us.

Reply:
Mmm ok. Do you want me to simplify the test, or keep it as-is since it’s there already? If the former, do I just pick one role to test a positive match?

Reply:
Just pick one of the roles to test it with. In the future I'm hoping we can add rule options so you could configure it that way.

Reply:
Done!